acceptance-tests/src/testFixtures/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -278,59 +276,40 @@ public void captureDebugArtifacts(final File artifactDir) {
-    private static final String BEACONRESTAPI_SECTION = "beaconrestapi";
-    private static final String DEPOSIT_SECTION = "deposit";
-    private static final String INTEROP_SECTION = "interop";
-    private static final String NODE_SECTION = "node";
-    private static final String VALIDATOR_SECTION = "validator";
     private static final String VALIDATORS_FILE_PATH = "/validators.yml";
-    private static final String OUTPUT_SECTION = "output";
-    private Map<String, Map<String, Object>> options = new HashMap<>();
+    private static final String P2P_PRIVATE_KEY_FILE_PATH = "/p2p-private-key.key";
     private static final int DEFAULT_VALIDATOR_COUNT = 64;
 
+    private Map<String, Object> configMap = new HashMap<>();
+
     private Optional<String> validatorKeys = Optional.empty();
     private Optional<GenesisStateConfig> genesisStateConfig = Optional.empty();
 
     public Config() {
-      final Map<String, Object> node = getSection(NODE_SECTION);
-      setNetworkMode("mock");
-      node.put("networkInterface", NetworkUtility.INADDR_ANY);
-      node.put("port", P2P_PORT);
-      node.put("discovery", "static");
-      node.put("constants", "minimal");
-
-      final Map<String, Object> interop = getSection(INTEROP_SECTION);
-      interop.put("genesisTime", 0);
-      interop.put("ownedValidatorStartIndex", 0);
-      interop.put("ownedValidatorCount", DEFAULT_VALIDATOR_COUNT);
-
-      final Map<String, Object> deposit = getSection(DEPOSIT_SECTION);
-      setDepositMode("test");
-      deposit.put("numValidators", DEFAULT_VALIDATOR_COUNT);
-
-      final Map<String, Object> beaconRestApi = getSection(BEACONRESTAPI_SECTION);
-      beaconRestApi.put("portNumber", REST_API_PORT);
-      beaconRestApi.put("enableSwagger", false);
-
-      final Map<String, Object> output = getSection(OUTPUT_SECTION);
-      output.put("transitionRecordDir", ARTIFACTS_PATH + "transitions/");
-
-      final Map<String, Object> database = getSection(DATABASE_SECTION);
-      database.put("dataDir", ARTIFACTS_PATH + "data/");
+      configMap.put("network", "minimal");
+      configMap.put("p2p-enabled", false);
+      configMap.put("p2p-discovery-enabled", false);
+      configMap.put("p2p-port", P2P_PORT);
+      configMap.put("p2p-advertised-port", P2P_PORT);
+      configMap.put("p2p-interface", NetworkUtility.INADDR_ANY);
+      configMap.put("x-interop-genesis-time", 0);
+      configMap.put("x-interop-owned-validator-start-index", 0);
+      configMap.put("x-interop-owned-validator-count", DEFAULT_VALIDATOR_COUNT);
+      configMap.put("x-interop-number-of-validators", DEFAULT_VALIDATOR_COUNT);
+      configMap.put("x-interop-enabled", true);
+      configMap.put("rest-api-enabled", true);
+      configMap.put("rest-api-port", REST_API_PORT);
+      configMap.put("rest-api-docs-enabled", false);
+      configMap.put("x-transaction-record-directory", ARTIFACTS_PATH + "transitions/");
+      configMap.put("data-path", ARTIFACTS_PATH);
```

Comment:
Seems cleaner to keep an explicit "data" directory here:
```suggestion
      configMap.put("data-path", ARTIFACTS_PATH + "data/");
```
But it looks like the various db-related paths  (`ARTIFACTS_DB_PATH` etc) will need to be updated.

Reply:
agreed